### ISSUE TITLE
Missing dependency for qemu-utils

### ISSUE THREAD

**Issue by kvaps** [OpenNebula/one#4947]
**Description**
Almost all tm and ds drivers [depends on qemu-img](https://github.com/OpenNebula/one/search?q=QEMU_IMG) installed on nodes, but it is not included to dependency list for `opennebula-node` package.

**To Reproduce**

Disable installing recommend packages:
```bash
# echo 'APT::Install-Recommends "0";\nAPT::Install-Suggests "0";' >> /etc/apt/apt.conf.d/01norecommend
```

Try install `opennebula-node`:
```bash
# apt install opennebula-node
Reading package lists... Done
Building dependency tree       
Reading state information... Done
The following additional packages will be installed:
  acl augeas-lenses augeas-tools cron dbus dmidecode gettext-base iproute2 ipset iptables ipxe-qemu
  ipxe-qemu-256k-compat-efi-roms iso-codes libaio1 libapparmor1 libargon2-1 libasound2 libasound2-data libaugeas0
  libboost-iostreams1.71.0 libboost-thread1.71.0 libbrlapi0.7 libbrotli1 libbsd0 libcacard0 libcap2 libcap2-bin
  libcbor0.6 libcryptsetup12 libcurl3-gnutls libdbus-1-3 libdevmapper1.02.1 libdrm-common libdrm2 libedit2 libelf1
  libepoxy0 libexpat1 libfdt1 libfido2-1 libfuse2 libgbm1 libgdbm-compat4 libgdbm6 libglib2.0-0 libgssapi-krb5-2
  libgstreamer-plugins-base1.0-0 libgstreamer1.0-0 libibverbs1 libicu66 libip4tc2 libip6tc2 libipset13 libiscsi7
  libjpeg-turbo8 libjpeg8 libjson-c4 libk5crypto3 libkeyutils1 libkmod2 libkrb5-3 libkrb5support0 libmnl0
  libnetfilter-conntrack3 libnfnetlink0 libnftnl11 libnghttp2-14 libnl-3-200 libnl-route-3-200 libnspr4 libnss3
  libnuma1 libopus0 liborc-0.4-0 libpam-systemd libparted2 libpcap0.8 libpci3 libpciaccess0 libpcsclite1
  libpixman-1-0 libpmem1 libpng16-16 libpolkit-agent-1-0 libpolkit-gobject-1-0 libpopt0 librados2 librbd1 librdmacm1
  librtmp1 libruby2.7 libslirp0 libspice-server1 libssh-4 libsystemd0 libusb-1.0-0 libusbredirparser1
  libvirglrenderer1 libvirt-clients libvirt-daemon libvirt-daemon-driver-qemu libvirt-daemon-system
  libvirt-daemon-system-systemd libvirt0 libwayland-server0 libx11-6 libx11-data libxau6 libxcb1 libxdmcp6 libxml2
  libxtables12 libyajl2 libyaml-0-2 logrotate netbase opennebula-common opennebula-common-onescape openssh-client
  pci.ids pciutils policykit-1 qemu-block-extra qemu-kvm qemu-system-common qemu-system-data qemu-system-x86 rake
  rsync ruby ruby-minitest ruby-net-telnet ruby-power-assert ruby-sqlite3 ruby-test-unit ruby-xmlrpc ruby2.7
  rubygems-integration seabios systemd systemd-container systemd-sysv systemd-timesyncd tzdata vlan
Suggested packages:
  augeas-doc anacron checksecurity default-mta | mail-transport-agent default-dbus-session-bus | dbus-session-bus
  iproute2-doc firewalld kmod nftables isoquery libasound2-plugins alsa-utils fuse gdbm-l10n krb5-doc krb5-user
  libvisual-0.4-plugins gstreamer1.0-tools opus-tools parted libparted-dev libparted-i18n pcscd
  gstreamer1.0-plugins-ugly libvirt-daemon-driver-lxc libvirt-daemon-driver-vbox libvirt-daemon-driver-xen
  libvirt-daemon-driver-storage-gluster libvirt-daemon-driver-storage-zfs numad apparmor auditd nfs-common open-iscsi
  pm-utils radvd systemtap zfsutils bsd-mailx | mailx keychain libpam-ssh monkeysphere ssh-askpass samba vde2
  openssh-server ri ruby-dev bundler
Recommended packages:
  libatm1 alsa-ucm-conf alsa-topology-conf libpam-cap dmsetup libglib2.0-data shared-mime-info xdg-user-dirs
  gstreamer1.0-plugins-base ibverbs-providers krb5-locales gstreamer1.0-plugins-good libxml2-utils netcat-openbsd
  libvirt-daemon-driver-storage-rbd dnsmasq-base parted lvm2 sudo openssh-server | ssh-server xauth qemu-system-gui
  qemu-utils ovmf cpu-checker zip sqlite3 fonts-lato libjs-jquery networkd-dispatcher libnss-mymachines
  libnss-systemd
The following NEW packages will be installed:
  acl augeas-lenses augeas-tools cron dbus dmidecode gettext-base iproute2 ipset iptables ipxe-qemu
  ipxe-qemu-256k-compat-efi-roms iso-codes libaio1 libapparmor1 libargon2-1 libasound2 libasound2-data libaugeas0
  libboost-iostreams1.71.0 libboost-thread1.71.0 libbrlapi0.7 libbrotli1 libbsd0 libcacard0 libcap2 libcap2-bin
  libcbor0.6 libcryptsetup12 libcurl3-gnutls libdbus-1-3 libdevmapper1.02.1 libdrm-common libdrm2 libedit2 libelf1
  libepoxy0 libexpat1 libfdt1 libfido2-1 libfuse2 libgbm1 libgdbm-compat4 libgdbm6 libglib2.0-0 libgssapi-krb5-2
  libgstreamer-plugins-base1.0-0 libgstreamer1.0-0 libibverbs1 libicu66 libip4tc2 libip6tc2 libipset13 libiscsi7
  libjpeg-turbo8 libjpeg8 libjson-c4 libk5crypto3 libkeyutils1 libkmod2 libkrb5-3 libkrb5support0 libmnl0
  libnetfilter-conntrack3 libnfnetlink0 libnftnl11 libnghttp2-14 libnl-3-200 libnl-route-3-200 libnspr4 libnss3
  libnuma1 libopus0 liborc-0.4-0 libpam-systemd libparted2 libpcap0.8 libpci3 libpciaccess0 libpcsclite1
  libpixman-1-0 libpmem1 libpng16-16 libpolkit-agent-1-0 libpolkit-gobject-1-0 libpopt0 librados2 librbd1 librdmacm1
  librtmp1 libruby2.7 libslirp0 libspice-server1 libssh-4 libusb-1.0-0 libusbredirparser1 libvirglrenderer1
  libvirt-clients libvirt-daemon libvirt-daemon-driver-qemu libvirt-daemon-system libvirt-daemon-system-systemd
  libvirt0 libwayland-server0 libx11-6 libx11-data libxau6 libxcb1 libxdmcp6 libxml2 libxtables12 libyajl2
  libyaml-0-2 logrotate netbase opennebula-common opennebula-common-onescape opennebula-node openssh-client pci.ids
  pciutils policykit-1 qemu-block-extra qemu-kvm qemu-system-common qemu-system-data qemu-system-x86 rake rsync ruby
  ruby-minitest ruby-net-telnet ruby-power-assert ruby-sqlite3 ruby-test-unit ruby-xmlrpc ruby2.7
  rubygems-integration seabios systemd systemd-container systemd-sysv systemd-timesyncd tzdata vlan
The following packages will be upgraded:
  libsystemd0
1 upgraded, 145 newly installed, 0 to remove and 4 not upgraded.
Need to get 53.4 MB of archives.
After this operation, 234 MB of additional disk space will be used.
Do you want to continue? [Y/n]
```

**Expected behavior**
apt will install qemu-utils package since it is vital for many drivers

**Details**
 - Affected Component: packages
 - Hypervisor: KVM
 - Version: 5.10 5.12

**Additional context**

## Progress Status
- [ ] Branch created 
- [ ] Code committed to development branch
- [ ] Testing - QA
- [ ] Documentation 
- [ ] Release notes - resolved issues, compatibility, known issues
- [ ] Code committed to upstream release/hotfix branches
- [ ] Documentation committed to upstream release/hotfix branches

**Comment by kvaps** [comment]
Since there is no publicly accessible packages repository anymore I'll just attach patch here:

[0001-B-4947-Missing-dependency-for-qemu-utils.patch.txt](https://github.com/OpenNebula/one/files/4819696/0001-B-4947-Missing-dependency-for-qemu-utils.patch.txt)

**Comment by kvaps** [comment]
Since there is no publicly accessible packages repository anymore I'll just leave my patch here:

[0001-B-4947-Missing-dependency-for-qemu-utils.patch.txt](https://github.com/OpenNebula/one/files/4819696/0001-B-4947-Missing-dependency-for-qemu-utils.patch.txt)

**Comment by vholer** [comment]
Thank you, @kvaps! Merged for master / one-5.12 (CE, EE).
